.github/workflows/ci.yml

```diff
@@ -49,3 +49,16 @@ jobs:
         run: |
           docker tag sync-team $REGISTRY/$REPOSITORY:latest
           docker push $REGISTRY/$REPOSITORY:latest
+
+      - name: Configure AWS credentials
+        if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: arn:aws:iam::890664054962:role/ci--rust-lang--sync-team
+          aws-region: us-west-1
```

Comment:
Uh, these are already configured up above? Seems like we don't need to do it twice.

Reply:
Gah, you're right. Will remove.